src/views/CardStack.js

```diff
@@ -199,30 +195,28 @@ class CardStack extends Component {
     animatedSubscribeValue(props.position);
   }
 
-  _reset(resetToIndex: number, velocity: number): void {
-    Animated.spring(this.props.position, {
+  _reset(resetToIndex: number, duration: number): void {
+    Animated.timing(this.props.position, {
       toValue: resetToIndex,
-      duration: ANIMATION_DURATION,
+      duration,
```

Comment:
Ah, I like this technique of controlling duration instead of the 'velocity'. Its a lot more clear what is happening